tedge_config/src/models/file_path.rs

```diff
@@ -41,9 +41,6 @@ impl AsRef<Path> for FilePath {
 
 impl std::fmt::Display for FilePath {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        match self.0.to_str() {
-            Some(s) => write!(f, "{}", s),
-            None => Err(std::fmt::Error),
-        }
+        write!(f, "{}", self.0.display())
```

Comment:
This isn't strictly needed for this PR, but I just noticed a nicer implementation here.